subnet/kube/kube.go

```diff
@@ -430,9 +438,24 @@ func (ksm *kubeSubnetManager) nodeToLease(n v1.Node) (l subnet.Lease, err error)
 		}
 		l.Attrs.BackendData = json.RawMessage(n.Annotations[ksm.annotations.BackendData])
 
-		_, cidr, err := net.ParseCIDR(n.Spec.PodCIDR)
-		if err != nil {
-			return l, err
+		cidr := new(net.IPNet)
+		if len(n.Spec.PodCIDRs) == 0 {
+			_, cidr, err = net.ParseCIDR(n.Spec.PodCIDR)
```

Comment:
Don't you need to check if this is ipv6 or ipv4?

Reply:
No because if IPv4 is enabled and no IPv4 CIDR is specified on the pod subnet Flannel should throw an error before the subnet manager starts.